docs/custom-flows/email-password.mdx

```diff
@@ -665,7 +664,7 @@ export default function SignUpScreen() {
                 const verify = await client.signUp.attemptEmailAddressVerification({
                     code
                 });
-                // user is created now set the session to active this never is not null.
+                // user isAp created now set the session to active this never is not null.
```

Comment:
"this never is not null" is confusing. is it always null, or is it never null? I'd think it's never null, because there will always be a sessionID

```suggestion
                // User is created. Now, set the session to active. session is never null.
```